charts/kyverno/Chart.yaml

```diff
@@ -26,5 +26,5 @@ annotations:
       url: https://kyverno.io/docs
   # valid kinds are: added, changed, deprecated, removed, fixed and security
   artifacthub.io/changes: |
-    - kind: added
-      description: make sigstore volume configurable
+    - kind: changed
```

Comment:
why are you changing an existing entry here ?

Reply:
I was not sure wether you want to keep the change list, so I just checked the last commit here https://github.com/kyverno/kyverno/commit/e0c9d8b0877492b1aa667fa470000e6cdd634078 and there is seemed to get replaced. But ok understoodd will just add the changelog